app/controllers/policy_classes_controller.rb

```diff
@@ -89,10 +96,10 @@ def ensure_can_assess_planning_application
   end
 
   def status
-    if params[:commit].downcase.match(/mark as complete/).present?
-      :complete
-    else
-      :in_assessment
-    end
+    commit_matches?(/mark as complete/) ? :complete : :in_assessment
+  end
+
+  def commit_matches?(regex)
```

Comment:
General comment.

The PolicyClassesController is getting a bit exciting. Would you consider using ServiceClasses, or other poros, to make them duller?

Reply:
Agreed. I've not made any changes for now, because I want to think a bit more about the best way to do it...but we definitely have a bit much going on here!